py/pysparkling/initializer.py

```diff
@@ -30,9 +35,21 @@ def __add_sparkling_jar_to_spark(sc):
             cl.addURL(url)
 
         # Add Sparkling water assembly JAR to executors
```

Comment:
I wrote this comment when writing this code, but I would change it to something like this:

> Add Sparkling Water Assembly JAR to Spark's file server so executors can fetch it when they need to use the dependency.

because we aren't actually adding any jars to executors at this phase